frontend/components/transaction/DepositBlock.vue

```diff
@@ -6,26 +6,30 @@
             the minimum amount to deposit in order to participate in the auction based on the bid you want to make. You
             can directly deposit the minimum required amount of DAI if you don't have sufficient deposits yet.
         </TextBlock>
-        <div class="text-base font-medium">
+        <div class="text-base font-medium text-gray-700 dark:text-gray-100">
             <div class="flex justify-between">
                 <span>Amount available in the wallet</span>
-                <format-currency v-if="walletDAI" :value="walletDAI" currency="DAI" />
+                <format-currency v-if="walletDai" :value="walletDai" currency="DAI" />
```

Comment:
I think we actually tried to always write `DAI` in uppercase, why did you decide to rename it everywhere?

Reply:
I renamed it because, if uppercased, the prop would change from `:minimum-deposit-dai="minimumDepositDai"` to `:minimum-deposit-d-a-i="minimumDepositDAI"` which I thought looked quite weird so I changed it, but if we try to keep it as uppercased everywhere I'll change it back 🙂

Reply:
Although this component prop prettification is not required by vue (you can keep it as `:minimumDepositDAI=""`) and it's not part of our linter setup (but your editor setup, most probably fixable adding `"vetur.format.enable": false` to your VSC settings), I see your point! Let's keep it as is.